windows/micropython.vcxproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
The usual methodology is to keep this up to date with the latest version and the project members that still have an older one need to change this file by hand prior to opening the project and exclude their project files from their commits. It's unfortunate that VS does not handle this scenario better, specially considering that newer versions can typically open an older solution and the changes occur automatically.